manifests/s/Sentinel/SentinelCLI/0.3.2/Sentinel.SentinelCLI.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Sentinel.SentinelCLI
+PackageVersion: 0.3.2
+InstallerType: burn

```

Comment:
```suggestion
InstallerType: portable
```

Reply:
With another installer type, Validation downloads the EXE, runs it, then deletes it (thinking it was the installer) - causing the ` Validation-No-Executables` label.